website/content/docs/operations/aws-oidc-provider.mdx

```diff
@@ -0,0 +1,361 @@
+---
```

Comment:
@tgross does this summary make sense? I was translating a support doc initially and was a little unsure of the terminology. This is my understanding based on your explanations.

_Nomad is the OIDC provider that gets registered with AWS IAM and subsequently trusted as such. It's responsible for creating workload identity tokens (JWTs) for tasks that the job then presents to AWS IAM for federated access to AWS resources. AWS in turn validates that token with Nomad (since it's the OIDC provider)._

Reply:
What is the difference between this workflow and auth-method create? This one shows how to configure AWS IAM to trust Nomad as an OIDC provider whereas the other configures the actual authentication Nomad provides as the OIDC provider?

Reply:
That summary LGTM.

Another way of looking at the difference is:
* In this workflow, AWS is configured to trust Nomad identities. That allows a Nomad workload to use AWS APIs.
* In the `auth-method create` workflow, Nomad is configured to trust AWS. That allows an AWS user to "login" to Nomad to use Nomad APIs.

Reply:
Ahhhh, that helps immensely

Reply:
Would a title like this make more sense then?
`Federate access to AWS with OIDC and Workload Identity`
Or even just
`Federate access to AWS with Nomad Workload Identity`

Reply:
Yeah that sounds great, especially the latter.